NameLayerMain/src/vg/civcraft/mc/namelayer/command/commands/RemoveMember.java

```diff
@@ -62,19 +61,16 @@ public boolean execute(CommandSender sender, String[] args) {
 		boolean allowed = false;
 		switch (toBeRemoved){ // depending on the type the executor wants to add the player to
 		case MEMBERS:
-			allowed = perm.isAccessible(t, PermissionType.MEMBERS);
+			allowed = gm.hasAccess(group.getName(), uuid, PermissionType.getPermission("MEMBERS"));
```

Comment:
Just a general note that struck me here; but hasAccess should have an overloaded method that accepts the "Group" object and doesn't repeat the lookup. As it stands this is really kind of bad behavior so to speak -- we retrieve the group in line 38 but gm.hasAccess will do it again, etc.